WordPress/gutenberg
# RNMobile - Replace Icon for media editing

`Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/2007

## Description
This is a proposal to add a `Replace` icon used in the mobile app.

Right now the Web editor has `Replace` as text in the toolbar but to have that on mobile is a bit more complicated due to the amount of space it'd take and also translations.

![ReplaceIcon](https://user-images.githubusercontent.com/4885740/76544908-21e25680-6489-11ea-9ca5-f743b033ab33.png)
We used to have this icon in two blocks but now we need it in another one so this is when we need to have it somewhere global and `@wordpress/icons` might just be the place.

This PR removes the duplicated icon and adds it to the icons library, now used in `Cover`, `Media & Text`, `Video`, and `Image` block.

## Screenshots 
| Image | Video |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/76546047-1a23b180-648b-11ea-80aa-dbb1d3fc4475.png" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/76545839-c2854600-648a-11ea-9886-0d315e1f92a8.png" width=250 />  |

| Media & Text |  Cover |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/76545875-cfa23500-648a-11ea-99ff-5c0c7af55e0f.png" width=250 />  | <img src="https://user-images.githubusercontent.com/4885740/76545896-daf56080-648a-11ea-9d02-9726cabffe33.png" width=250 />  |

## How has this been tested?
- Storybook
- On the app
  - Lunch the app with metro running
  - Add blocks: Video, Media & Text, Image and Cover
  - Set a media in all blocks
  - **Expect** the replace media icon to be the same for all blocks.

## Types of changes
New icon

## Checklist:
